src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -10,33 +10,35 @@
 
 @Controller
 public class LocalTimeService {
-	
-	private LocalDateTime localDateTime = LocalDateTime.now();
-	
+
```

Comment:
Jeszcze tylko powycinaj puste linie i przepchnę pull requesta

Reply:
Zerknij czy teraz wszystko jest OK.

Reply:
Jeśli nie robisz tego o co Cię proszę to nie jest OK. Wytnij puste linie wszędzie